src/components/TextInputWithCurrencySymbol/BaseTextInputWithCurrencySymbol.tsx

```diff
@@ -58,22 +61,26 @@ function BaseTextInputWithCurrencySymbol(
             }}
             onKeyPress={onKeyPress}
             style={styles.pr1}
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...rest}
         />
     );
 
     if (isCurrencySymbolLTR) {
         return (
             <>
-                {currencySymbolButton}
+                {!hideCurrencySymbol && currencySymbolButton}
```

Comment:
Should we update the definition of 
```
 const currencySymbolButton = hideCurrencySymbol ? undefined : (
        <CurrencySymbolButton
            currencySymbol={currencySymbol ?? ''}
            onCurrencyButtonPress={onCurrencyButtonPress}
            isCurrencyPressable={isCurrencyPressable}
        />
    );

```

And remain this code to avoid duplicated code